packages/dds/tree/src/simple-tree/api/create.ts

```diff
@@ -0,0 +1,179 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IFluidHandle } from "@fluidframework/core-interfaces";
+import { assert } from "@fluidframework/core-utils/internal";
+
+import type { ITreeCursorSynchronous, SchemaAndPolicy } from "../../core/index.js";
+import { fail } from "../../util/index.js";
+import type {
+	TreeLeafValue,
+	ImplicitFieldSchema,
+	InsertableTreeFieldFromImplicitField,
+	TreeFieldFromImplicitField,
+	FieldSchema,
+	FieldKind,
+} from "../schemaTypes.js";
+import type { Unhydrated } from "../core/index.js";
+import {
+	cursorForMapTreeNode,
+	defaultSchemaPolicy,
+	FieldKinds,
+	intoStoredSchema,
+	mapTreeFromCursor,
+	type NodeKeyManager,
+} from "../../feature-libraries/index.js";
+import { getOrCreateNodeFromFlexTreeNode, type InsertableContent } from "../proxies.js";
+import { getOrCreateMapTreeNode, isFieldInSchema } from "../../feature-libraries/index.js";
+import { toFlexSchema } from "../toFlexSchema.js";
+import { inSchemaOrThrow, mapTreeFromNodeData } from "../toMapTree.js";
+import {
+	applySchemaToParserOptions,
+	cursorFromVerbose,
+	type ParseOptions,
+	type VerboseTree,
+	type VerboseTreeNode,
+} from "./verboseTree.js";
+
+/**
+ * Construct tree content compatible with a field defined by the provided `schema`.
```

Comment:
Nit (was a bit hard to parse):
```suggestion
 * Construct tree content that is compatible with the field defined by the provided `schema`.
```